samples/helloworld/README.md

```diff
@@ -508,6 +510,18 @@ ninja
 ninja run
 ```
 
+### Mixing LVI Mitigation
+
+You can use a different LVI mitigation method to build your app than Open Enclave, however additional cmake configuration may be required. See the table below for all possible combinations.
+
+
+| OE    | client | Example cmake command                                                                                   | Additional Requirement                                                                                                   |
+|-------|-----------|---------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
```

Comment:
what happens if someone specifies _DLVI_MITIGATION=ControlFlow? Can you default to ControlFlow_Gnu.
Can you also include a note in bold? OE release packages are built with Controlflow Gnu. 
OE built with ControlFlow_Clang is only available when contributors build from source

Reply:
Changed doc